tests/installation/partitioning_lvm.pm

```diff
@@ -100,15 +100,15 @@ sub run {
             assert_screen "inst-select-root-disk";
             send_key 'alt-n';
         }
-        if (leap_version_at_least('15.0')) {
+        if (leap_version_at_least('15.0') || is_tumbleweed()) {
             send_key 'alt-a';
         }
         else {
             send_key 'alt-e';
         }
         assert_screen "inst-partitioning-lvm-enabled";
         if (get_var("ENCRYPT")) {
-            if (leap_version_at_least('15.0')) {
+            if (leap_version_at_least('15.0') || is_tumbleweed()) {
```

Comment:
Same as above: the function leap_version_at_least should be fixed

Reply:
I don't know what you mean by that. How should the function be *fixed*?

Reply:
If Tumbelweed is considered as always the latest version I'd expect a function like `leap_version_at_least` to return `true` for Tumbleweed.

Reply:
@okurz, I missed your questions. I mean what @cwh42 states.

So, is it wanted that is_leap_at_least does not return true for TW?

Reply:
yes, this is wanted. I know that the specification of `sle_version_at_least` and `leap_version_at_least` differs here and you can try to change that. So what can be *fixed* is that the specification differs but each function on its own is acting according to its specification. 

@cwh42 It is correct to state that Tumbleweed should have the latest version of all packages before any other version but it is still a different distribution. Please try to avoid the function `is_tumbleweed` at all. If the submission and acceptance policy would be strict we would have less of a problem here: TW would have storage-ng first, we fix the test for that version and run the old test code for any other released software and then extend for newer versions of Leap and SLE. What we have to live with for the moment is a mix. I recommend the following:

```perl
if (!(is_leap && !leap_version_at_least('15.0'))) {
    # new code
}
else {
    # old code
}
```

Reply:
Not sure how often cases like this occur, but it sounds really like the most complicated solution possible.
I think every test should fit for factory. Then special cases can be introduced for older versions. It would make life easier to have someting like:
```perl
if (leap_version_and_older('15.0')) {
    # code
}
elsif (sle_version_and_older('12.0')) {
    # other code
}
else {
    # code works for factory
}
```